### DISCUSSION TITLE
SearXNG not displaying any results with Engine timeout

### DISCUSSION THREAD

**Discussion by ncarvalho99** [searxng/searxng#3732] [Q&A] [answered]
Hello, i installed SearXNG on my CasaOS homelab using Portainer using this template:
`version: '3.9'
services:
  searxng:
    image: searxng/searxng
    container_name: SearXNG
    mem_limit: 8g
    cpu_shares: 2048
    security_opt:
      - no-new-privileges:true
    volumes:
      - /volume1/docker/searxng:/etc/searxng:rw 
    restart: on-failure:5
    ports:
      - 5147:8080`
It was as sucess, but when i try to search something, i get the following result:
![image](https://github.com/user-attachments/assets/9d2a2f1d-01de-4ae1-a24f-9c9d74aed60d)

On my CasaOS homelab i have PiHole configured has my DNS Server and i'm using Cloudflare-web to have access to my instances out of home, i don't know if any of this is causing some error to occour. 

Is there any way i can fix it ?

**Comment by return42** [comment, marked as answer]
This is the SearXNG core project,

> CasaOS homelab using Portainer ... Cloudflare-web

 we do not support any of the components you use.  May your questions find some interest on [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org).
